recipe/patch_yaml/cppyy-cling.yaml

```
@@ -0,0 +1,10 @@
+if:
+  name: cppyy-cling
+  subdir_in: ["osx-64", "osx-arm64"]
+  version: 6.30.0
+  build_number_in: [0, 1, 2]
+  timestamp_lt: 1726941093000
+then:
+  - replace_depends:
+      old: libcxx >=14.0.6
+      new: ${old},<18
```

```suggestion
  - tighten_depends:
      name: libcxx
      upper_bound: 18
```

i think this is better spelled like the above.

---

You can keep this in mind for future patches.